### ISSUE TITLE
Narrator/NVDA not announcing “Filter by” label name when focus lands on the first filter Checkbox.: A11y_Awesome AZD_ Template library_Search_Screen reader

### ISSUE THREAD

**Issue by Harika1702** [Azure/awesome-azd#345]
### Tags
#A11yTCS; #A11ySev3; #A11yMAS; #WCP; #ChromiumEdge; #FTP; #Win11; #awesome-azd-wcp; #Feature_AwesomeAZD_Web_Mar2024; #WCAG1.3.1; #Info and Relationship; #AILimited; #DesktopWeb; #Element:Label; #ScreenReader; #NVDA; #Narrator; #STP; #Chrome; 

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."
### Actual Result:
Narrator/NVDA not announcing “Filter by” label name when focus lands on the first filter Checkbox.
### Expected Results:
Narrator/NVDA should announce “Filter by” label name when focus lands on the first filter Checkbox.
### Environment Details:
Application: Awesome AZD
URL: https://azure.github.io/awesome-azd/
Chromium Edge: Version 123.0.2420.20 
Chrome Version 122.0.6261.131 (Official Build) (64-bit)
Narrator/NVDA Version 2023.3.3
Windows 11 Enterprise Version 23H2 OS build 22631.2050.
### Repro Steps: 
1. Turn on narrator/NVDA.
2. Hit this URL: https://azure.github.io/awesome-azd/.
3. Tab till “Filter by” section.
4. Tab till first filter checkbox.
5. Now verify the issue that Narrator/NVDA not announcing “Filter by” label name when focus lands on the first filter Checkbox or not.
### User Impact:
Screen reader users will not be able to know under which section they are navigating if the label name is not announced.

### Attachment:
<img width="960" alt="345_A11y_Code snippet" src="https://github.com/Azure/awesome-azd/assets/100270074/b4208684-df20-478d-bf2d-f1ec53bbd0f4">

https://github.com/Azure/awesome-azd/assets/100270074/c624ba76-5122-4a6f-b39f-a7033b7b929c

https://github.com/Azure/awesome-azd/assets/100270074/a873d129-619a-4661-aefa-6f72066fde37

**Comment by hemarina** [comment]
Closing this issue as we made a whole update on the site, and this issue is not update to date with existing site. @Harika1702 Feel free to reopen this issue if needed.
